raft/test/raft_test.cpp

```diff
@@ -585,15 +584,15 @@ namespace bzn
         resp.clear();
 
         // send invalid entry. peer should return false and not move back past commit index
-        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 0, 0, 0, 0, entry);
+        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 1, 0, 0, 0, entry);
```

Comment:
To test rejecting invalid entries, this should be at the next log index after the prev one, so commit_index 3, prev_index 2. It gets rejected because it's not from the current term?